bibliography.bib

```diff
@@ -8224,3 +8224,612 @@ @InProceedings {ChiKimStaKim2021humanoids
+    title        = {{SWIRL}: a sequential windowed inverse reinforcement learning algorithm for robot tasks with delayed rewards},
+    author       = Krishnan_S # and # Garg_A # and # Liaw_R # and Thananjeyan_B # and # Miller_L # and # Pokorny_FT # and # Goldberg_K,
+    journal      = IJRR,
+    series       = {},
+    volume       = {38},
+    number       = {2--3},
+    year         = {2019},
+    pages        = {126--145},
+    doi          = {10.1177/0278364918784350},
+    note         = {},
+    annote       = {},
+}
+
+@Article {LiGauGro2016SI,
+    title        = {Turing learning: a metric-free approach to inferring behavior and its application to swarms},
+    author       = Li_W # and # Gauci_M # and # Gross_R,
+    journal      = SI,
+    series       = {},
+    volume       = {10},
+    number       = {},
+    year         = {2016},
+    pages        = {211--243},
+    doi          = {10.1007/s11721-016-0126-1},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {AlhAbdHau2022ants,
+    title        = {Automatic Extraction of Understandable Controllers from Video Observations of Swarm Behaviors},
+    author       = Alharthi_K # and # Abdallah_ZS # and # Hauert_S,
+    booktitle    = ants_2022,
+    editor       = Dorigo_M # and # Hamann_H # and # LopezIbanez_M # and # GarciaNieto_J # and # Engelbrecht_AP # and # Pinciroli_C # and # Strobel_V # and # CamachoVillalon_C,
+    series       = {},
+    volume       = {13491},
+    number       = {},
+    year         = {2022},
+    pages        = {41--53},
+    publisher    = pub_SP,
+    address      = Cham_CHE,
+    doi          = {10.1007/978-3-031-20176-9_4},
+    note         = {},
+    annote       = {},
+}
+
+@TechReport {LegGarKuc-etal2022techrep,
+    title        = {{Tycho}: a robust, {ROS}-based tracking system for robot swarms},
+    author       = LegardaHerranz_G # and # GarzonRamos_D # and # Kuckling_J # and # Kegeleirs_M # and # Birattari_M,
+    institution  = ins_IRIDIA_ULB_BEL,
+    address      = Brussels_BEL,
+    year         = {2022},
+    number       = {TR/IRIDIA/2022-009},
+    note         = {},
+    annote       = {},
+}
+
+@Article{HasLigBir2022IEEETEC,
+    title        = {Towards the automatic design of automatic methods for the design of robot swarms},
+    author       = Hasselmann_K # and # Ligot_A # and # Birattari_M,
+    journal      = SWEVO,
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {},
+    pages        = {},
+    doi          = {},
+    note         = {submitted for publication},
+    annote       = {},
+}
+
+@Misc {Gct2018wiki,
+    title        = {Overo Extension},
+    author       = {{GCtronic}},
+    howpublished = {\url{https://www.gctronic.com/doc/index.php/Overo_Extension}},
+    year         = {2018},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {ZieMaaBagDey2008aaai,
+    title        = {Maximum Entropy Inverse Reinforcement Learning},
```

Comment:
Titles should be in sentence case, please check also other entries, I spotted a similar mistake for AlhAbdHau2022ants